plugins/modules/grafana_folder.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright: (c) 2019, Rémi REY (@rrey)
+
+from __future__ import absolute_import, division, print_function
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'metadata_version': '1.1'
+}
+
+DOCUMENTATION = '''
+---
+module: grafana_folder
+author:
+  - Rémi REY (@rrey)
+version_added: "2.10"
+short_description: Manage Grafana Folders
+description:
+  - Create/update/delete Grafana Folders through the Folders API.
+  - Also allows to add members in the team (if members exists).
+  - The Folders API is only available starting Grafana 5 and the module will fail if the server version is lower than version 5.
+options:
+  url:
+    description:
+      - The Grafana URL.
+    required: true
+    type: str
+  name:
+    description:
+      - The title of the Grafana Folder.
+    required: true
+    type: str
+    aliases: ["title"]
```

Comment:
minor: to be consistent with other `aliases`
```suggestion
    aliases: [ title ]
```